src/hooks/queries/useBookInfo/types.ts

```diff
@@ -0,0 +1,94 @@
+interface VolumeInfo {
```

Comment:
[?] Do we have access to all this information about the books? 😮

Reply:
yeah haha pretty cool right?